src/Makefile.am

```diff
@@ -31,8 +31,30 @@ LOADABLE_MODULES_SOURCES = \
 	LoadableModules.cc \
 	LoadableModules.h
 
-SUBDIRS	= mem base anyp helper dns ftp parser comm eui acl format clients sbuf servers fs repl store DiskIO proxyp
-DIST_SUBDIRS = mem base anyp helper dns ftp parser comm eui acl format clients sbuf servers fs repl store DiskIO proxyp
+SRC_DIRS = \
```

Comment:
Please undo this variable addition. The whole sequence does need cleaning up. But not as part of this PR.

Reply:
commit 0575ca35a9